### PR TITLE
add cc to Mail preview

### PR DESCRIPTION
**Pull request by ValentinHasler**

### CODE REVIEW

File: web/package.json [hunk 1]
```diff
@@ -57,5 +57,7 @@
       "<rootDir>/tests/__mocks__/setupTests.js"
     ]
   },
-  "dependencies": {}
+  "dependencies": {
+    "styled-jsx": "3.2.4"
+  }
```
Comment (sventschui) [inline web/package.json]:
```suggestion
  "dependencies": {}
```

File: server/src/graphql/index.mjs [hunk 2]
```diff
@@ -135,6 +135,7 @@ const typeDefs = gql`
     id: ID!
     from: SenderRecipient
     to: [SenderRecipient]!
+    cc: [SenderRecipient]
```
Comment (sventschui) [inline server/src/graphql/index.mjs]:
```suggestion
    cc: [SenderRecipient]!
```

There will always be at least an empty array present

**Review by sventschui — COMMENTED** [review]
Thanks for the contribution. Just two small nitpicks, looks good otherwise

File: server/src/graphql/index.mjs (hunk 2 above)
Comment (ValentinHasler) [reply server/src/graphql/index.mjs]: done

**Pull request merged**